src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    if (typeof compareFunction !== 'function') {
+      // eslint-disable-next-line no-param-reassign
+      compareFunction = (a, b) => {
+        if (a.toString() < b.toString()) {
+          return -1;
+        }
+
+        if (a.toString() > b.toString()) {
+          return 1;
+        }
+
+        return 0;
+      };
+    }
```

Comment:
just specify the default value for the argument

```suggestion
  [].__proto__.sort2 = function (compareFunction = ...) {
    }
```